src/styles/global.css

```diff
@@ -0,0 +1,24 @@
+@import "/src/assets/fonts/inter.css";
+@import "/src/styles/ui-colors.css";
+
+:root {
+    --max-width: 100%;
+    --border-radius: 12px;
+    --font: Inter;
+}
+
+* {
```

Comment:
I would recommend to put the styling for * on top, as it weights higher than `:root`

Reply:
there is nothing higher than the root element!

Reply:
normally you put the * to the top as it is has the lowest weight